### DISCUSSION TITLE
Navigation widget

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#2712] [Ideas / Feature Requests]
I found [this](https://opentiny.design/tiny-ng/components/leftmenu) ready to use navigation widget is very convenient,if we could use them with nicegui,or if nicegui could import such elements It is also based on vue

**Comment by rodja** [comment]
The lib looks like it's Angular, not Vue. Maybe there are other libs with similar features (ideally already based on Quasar/Vue)?
